README.md

```diff
@@ -1501,6 +1501,7 @@ A registry allows you to publish your Rust libraries as crate packages, to share
   * [Learning Rust With Entirely Too Many Linked Lists](http://cglab.ca/~abeinges/blah/too-many-lists/book/) — in-depth exploration of Rust's memory management rules, through implementing a few different types of list structures.
   * [Rust by Example](https://doc.rust-lang.org/rust-by-example/)
   * [Rust Cookbook](https://rust-lang-nursery.github.io/rust-cookbook/) — A collection of simple examples that demonstrate good practices to accomplish common programming tasks, using the crates of the Rust ecosystem.
+  * [Rust Online Courses at Classpert](https://classpert.com/rust-programming) — A list of Rust online courses from Classpert Online Course Search
```

Comment:
These are almost entirely paid courses, so should be marked as such. "(paid)" at the end of the line would be good.

Reply:
Done! Thank you very much